tests/system/action/meeting/test_clone.py

```diff
@@ -519,28 +519,6 @@ def test_permissions_oml_can_manage(self) -> None:
             "meeting/2", {"is_active_in_organization_id": 1, "committee_id": 2}
         )
 
-    def test_permissions_missing_meeting_committee_permission(self) -> None:
```

Comment:
Also leave this test in and check instead that it suceeds to "prove" that the changes work

Reply:
Add a test, which shows, that the permission check is in place.